cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -337,113 +335,109 @@ instance ( ConvertRawHash blk
-        ChainDB.ClosedDB immTip tip' -> \_o ->
+        ChainDB.ClosedDB immTip tip' ->
           "Closed db with immutable tip at " <> renderPointAsPhrase immTip <>
           " and tip " <> renderPointAsPhrase tip'
-        ChainDB.OpenedImmutableDB immTip chunk -> \_o ->
+        ChainDB.OpenedImmutableDB immTip chunk ->
           "Opened imm db with immutable tip at " <> renderPointAsPhrase immTip <>
           " and chunk " <> showT chunk
-        ChainDB.OpenedVolatileDB -> \_o -> "Opened vol db"
-        ChainDB.OpenedLgrDB -> \_o -> "Opened lgr db"
+        ChainDB.OpenedVolatileDB ->  "Opened vol db"
+        ChainDB.OpenedLgrDB ->  "Opened lgr db"
       ChainDB.TraceReaderEvent ev -> case ev of
-        ChainDB.NewReader -> \_o -> "New reader was created"
-        ChainDB.ReaderNoLongerInMem _ -> \_o -> "ReaderNoLongerInMem"
-        ChainDB.ReaderSwitchToMem _ _ -> \_o -> "ReaderSwitchToMem"
-        ChainDB.ReaderNewImmIterator _ _ -> \_o -> "ReaderNewImmIterator"
+        ChainDB.NewReader ->  "New reader was created"
+        ChainDB.ReaderNoLongerInMem _ ->  "ReaderNoLongerInMem"
+        ChainDB.ReaderSwitchToMem _ _ ->  "ReaderSwitchToMem"
+        ChainDB.ReaderNewImmIterator _ _ ->  "ReaderNewImmIterator"
       ChainDB.TraceInitChainSelEvent ev -> case ev of
-        ChainDB.InitChainSelValidation _ -> \_o -> "InitChainSelValidation"
+        ChainDB.InitChainSelValidation _ ->  "InitChainSelValidation"
       ChainDB.TraceIteratorEvent ev -> case ev of
-        ChainDB.UnknownRangeRequested _ -> \_o -> "UnknownRangeRequested"
-        ChainDB.BlockMissingFromVolatileDB _ -> \_o -> "BlockMissingFromVolatileDB"
-        ChainDB.StreamFromImmutableDB _ _ -> \_o -> "StreamFromImmutableDB"
-        ChainDB.StreamFromBoth _ _ _ -> \_o -> "StreamFromBoth"
-        ChainDB.StreamFromVolatileDB _ _ _ -> \_o -> "StreamFromVolatileDB"
-        ChainDB.BlockWasCopiedToImmutableDB _ -> \_o -> "BlockWasCopiedToImmutableDB"
-        ChainDB.BlockGCedFromVolatileDB _ -> \_o -> "BlockGCedFromVolatileDB"
-        ChainDB.SwitchBackToVolatileDB -> \_o -> "SwitchBackToVolatileDB"
-      ChainDB.TraceImmutableDBEvent _ev -> \_o -> "TraceImmutableDBEvent"
-      ChainDB.TraceVolatileDBEvent _ev -> \_o -> "TraceVolatileDBEvent"
+        ChainDB.UnknownRangeRequested _ ->  "UnknownRangeRequested"
+        ChainDB.BlockMissingFromVolatileDB _ ->  "BlockMissingFromVolatileDB"
+        ChainDB.StreamFromImmutableDB _ _ ->  "StreamFromImmutableDB"
+        ChainDB.StreamFromBoth _ _ _ ->  "StreamFromBoth"
+        ChainDB.StreamFromVolatileDB _ _ _ ->  "StreamFromVolatileDB"
+        ChainDB.BlockWasCopiedToImmutableDB _ ->  "BlockWasCopiedToImmutableDB"
+        ChainDB.BlockGCedFromVolatileDB _ ->  "BlockGCedFromVolatileDB"
+        ChainDB.SwitchBackToVolatileDB ->  "SwitchBackToVolatileDB"
+      ChainDB.TraceImmutableDBEvent _ev ->  "TraceImmutableDBEvent"
+      ChainDB.TraceVolatileDBEvent _ev ->  "TraceVolatileDBEvent"
```

Comment:
Strictly speaking, #1123 is also tracing these two subevents.

But since I'm not a fan of the duplication between this text-based instance and the structured instance below, I don't feel strongly about this.

Question: which of the two is the default (or the one the default config picks)? I hope the structured one.

Reply:
We currently default to the unstructured text.